src/test/java/org/erp/school/SchoolApplicationDevTests.java

```diff
@@ -1,34 +0,0 @@
-package org.erp.school;
-
-import org.junit.jupiter.api.Test;
-import org.junit.jupiter.api.extension.ExtendWith;
-import org.mockito.junit.jupiter.MockitoExtension;
-import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
-import org.springframework.boot.test.context.SpringBootTest;
-import org.springframework.http.MediaType;
-import org.springframework.test.context.ActiveProfiles;
-import org.springframework.test.web.servlet.MockMvc;
-import org.springframework.test.web.servlet.MvcResult;
-import org.springframework.test.web.servlet.request.MockMvcRequestBuilders;
-
-import static org.junit.jupiter.api.Assertions.assertEquals;
-
-@ExtendWith(MockitoExtension.class)
-@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.MOCK, classes = SchoolApplication.class)
-@AutoConfigureMockMvc
-@ActiveProfiles("dev")
-class SchoolApplicationDevTests {
```

Comment:
It is important to leave this class here. The SchoolApplicationDevTests and SchoolApplicationTeste are used in the Travis to check if the application can be started. 

I have cases that the code was compiling ok, but build was failed because it could'nt start the app.

Could you rollback these two files, please?

Reply:
@patricksan can we have a call about this one? I would like to know how exactly Travis checks this part.

Reply:
We can call to discuss, but just to let you know in front, Travis will execute all the tests.
These two tests will start the application and check if spring context is properly loaded for profile `dev` and `stage`.

This is done via:

`$ mvn clean install`

Reply:
@patricksan that's what I thought. You don't need the home controller endpoint for those, neither do you need related tests. If you are just checking build success then it already does it when building. 

If you still wish to have some sort of confirmation that the application is running it can be done through the actuator endpoint and potentially we can write tests for that. In fact, I think I will write those, but still, I think that completely unnecessary as we will build up a test base sooner or later.

Reply:
@Aurrix here there are two points:
1. Remove the Home Controller
2. Test if the application spring context is loaded correctly.

At the first point we have a real controller test, then we can delete the current test and the Home controller.
Until there, we need the test to check if spring context is correctly loaded during the test. If you remove this class, there is no way today to check if the application is starting.

But your statement is correct that we will remove both of them but only when there is proper test.

If your PR creates a test for your controller, you can delete HomeController + SchoolApplicationDevTests. If not, please, still let it there for Travis.